checks/binary_artifact.go

```diff
@@ -22,22 +22,31 @@ import (
 	"github.com/h2non/filetype/types"
 
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged